virtual/cluster_up.sh

```diff
@@ -1,37 +1,96 @@
 #!/bin/bash
-set -x
+set -ex
 
 # Start vagrant via libvirt - set up the VMs
 vagrant up --no-parallel --provider=libvirt
 
+# Show the running VMs
+virsh list
+
 cd ..
 
+# Create the config for deepops servers (and use the virtual inventory)
+cp -r config.example/ config/
```

Comment:
It would be great if we could maintain two inventories at the same time (e.g. `./hosts.yml` and `./virtual/hosts.yml`). Then, we could use deepops for multiple clusters at the same time.

I think I nice way to support that would be to collapse all the variables in `./config/group_vars/*` into a single, YAML, config file.

But I also think we could do this even now by changing this line to `cp -r config.example/ virtual/config/`, and use `-i virtual/config/inventory` in the rest of the ansible commands in this file.

What do you think?

Reply:
I was thinking of that as well, actually. Totally agree.